addons/website/views/res_config_settings_views.xml

```diff
@@ -4,7 +4,8 @@
         <field name="name">res.config.settings.view.form.inherit.website</field>
         <field name="model">res.config.settings</field>
         <field name="priority" eval="20"/>
-        <field name="inherit_id" ref="base.res_config_settings_view_form"/>
+        <field name="inherit_id" ref="auth_signup.res_config_settings_view_form"/>
+        <field name="groups_id" eval="[(4, ref('website.group_website_designer'), 0)]"/>
```

Comment:
This shouldn't be conditioned based on groups (at least the hiding part, see below).

If you have website installed, the `Customer Account` settings in the `General Settings` tab has no sense/meaning anymore (its why it is hidden and replace by a field on the website settings).

You can probably just extract the
```xml
<xpath expr="//div[@id='login_documents']" position="attributes">
    <attribute name="invisible">1</attribute>
</xpath>
```
part in another view without `groups_id` to keep this field hidden to everyone.

See https://watch.screencastify.com/v/adWaL2dlY4upbEZZO4OV